CONTRIBUTING.md

```diff
@@ -40,6 +40,8 @@ go build ./cmd/prometheus/
 make test         # Make sure all the tests pass before you commit and push :)
 ```
 
+We use `golangci-lint`[https://github.com/golangci/golangci-lint] for linting the code. If it reports an issue and you think that it deserves an exception, you can add a special comment `//nolint:linter1[,linter2,...]` before the offending line. Use this sparingly though, fixing the code to comply with the linter's recommendation is in general the preferred course of action.
```

Comment:
I feel that `it deserves an exception` isn't too clear. Something along the lines of "if the linter's warning needs to be disregarded or is a false positive", perhaps?